the-collab-lab/tcl-70-smart-shopping-list
# Fill About page with Info and links

## Description

The About page was empty, but now uses a carousel to display different info about the project, the developers, and about The Collab Lab. Styling is not yet complete though.

## Related Issue

closes #75 

## Acceptance Criteria

- [x] Write info about the app
- [x] Write info about TCL
- [x] Add links to the Github (code/repo)
- [x] Add link to TCL site
- [x] Create 4 cards on the page that will be used for the developer profiles 
- [x] Use MUI Avatar for the profile pics (placeholder image for now)
- [x] Add links/use icons as placeholders to any social media/github (github, linkedin, twitter, portfolio etc)

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|  ✓    | :sparkles: Enhancements     |

## Updates

### Before

![Screenshot 2024-04-04 at 1 58 44 AM](https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/110392224/6b473bc2-295c-46cc-8f96-7865737f3898)

### After

<img width="1189" alt="Screenshot 2024-04-04 at 4 39 39 PM" src="https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/110392224/311e5351-8354-45fc-aa33-34112b74dcc9">
<img width="862" alt="Screenshot 2024-04-04 at 4 40 05 PM" src="https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/110392224/9e010a71-9bff-4969-b984-8a84fafbf2b9">
<img width="1133" alt="Screenshot 2024-04-04 at 4 39 51 PM" src="https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/110392224/d36351cd-1d00-45c4-b6c2-f9343eeedd1e">
<img width="872" alt="Screenshot 2024-04-04 at 4 40 00 PM" src="https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/110392224/a52b855e-004b-4969-ba8e-252fb0600336">
<img width="1152" alt="Screenshot 2024-04-04 at 4 39 46 PM" src="https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/110392224/ca7fe468-fbf8-4fed-aa3d-b8c62c9678a9">
<img width="854" alt="Screenshot 2024-04-04 at 4 40 14 PM" src="https://github.com/the-collab-lab/tcl-70-smart-shopping-list/assets/110392224/6d8fea09-aac2-4b01-84f5-ae300a856083">

## Testing Steps / QA Criteria

- From your terminal, pull down this branch with `git pull origin gl-about-page-info` and check that branch out with `git checkout gl-about-page-info`
- Run `npm start` in your terminal
- Navigate to the About page
